tests/test_reader.py

```diff
@@ -414,9 +414,8 @@ def test_get_page_mode(src, expected):
 
 
 def test_read_empty():
-    with pytest.raises(PdfReadError) as exc:
+    with pytest.raises(EmptyFileError):
         PdfReader(io.BytesIO())
-    assert exc.value.args[0] == "Cannot read an empty file"
```

Comment:
Please leave the comparison in the test. I like the fact that users can now rely on the more specific exception, but I also want to see that the message is fine

Reply:
done